src/dev/build/tasks/nodejs/download_node_builds_task.ts

```diff
@@ -30,16 +30,25 @@
 
 import { download, GlobalTask } from '../../lib';
 import { getNodeShasums } from './node_shasums';
-import { getNodeDownloadInfo, getRequiredVersion } from './node_download_info';
+import {
+  getNodeDownloadInfo,
+  getNodeVersionDownloadInfo,
+  getRequiredVersion,
+  NODE14_FALLBACK_VERSION,
+} from './node_download_info';
 
 export const DownloadNodeBuilds: GlobalTask = {
   global: true,
   description: 'Downloading node.js builds for all platforms',
   async run(config, log) {
     const requiredNodeVersion = getRequiredVersion(config);
     const shasums = await getNodeShasums(log, requiredNodeVersion);
-    await Promise.all(
-      config.getTargetPlatforms().map(async (platform) => {
+
+    // ToDo [NODE14]: Remove this Node.js 14 fallback download
```

Comment:
I wonder if there would be a slight improvement to not even attempt to download node 14 if the the current node version fails as well (for platform availability reasons).

Reply:
At this stage we won't know since it is a redistributable package that we build here.